commander_frontend/script.js

```diff
@@ -1,28 +1,65 @@
+//Imports
+import { Teste } from "./modules/Teste.js";
+import { Item } from "./modules/Item.js";
+import { createItem } from "./modules/CreateElements.js";
 
+//Send Button
```

Comment:
Keep your code clean, comments aren't needed. Well written code doesn't need comments. It's good to organize your steps, but please delete it before PR.